composer.json

```diff
@@ -25,7 +25,8 @@
         "psr/cache":        "^1.0",
         "predis/predis":    "^1.0",
         "doctrine/cache":   "^1.3",
-        "league/flysystem": "^1.0"
+        "league/flysystem": "^1.0",
+        "symfony/options-resolver": "^3.0"
```

Comment:
`2.7|3.0`, Right?

The option resolver was introduced in 2.6 but we should use 2.7 because it is LTS.